### PR TITLE
Fizz buzz

### PR DESCRIPTION
**Pull request by JRSorem**
For some reason, it also thinks I updated MOVIES.md but it's the same file as in the master

### CODE REVIEW

**Comment by sleddog** [comment]
@JRSorem can you add the java source code vs the jar file?

File: projects/fizzbuzz/java/FizzBuzz.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            //If number is divisible by 5
+            if (i%5 == 0)
+            {
+                System.out.print ("Buzz");
+            }
+            //If not divisible by 3 or 5
+            if (i%3 != 0 && i%5 != 0)
+            {
+                System.out.print (i);
+            }
```
Comment (NickHager) [inline projects/fizzbuzz/java/FizzBuzz.java]: Solution should work as is. However, there is one way to improve efficiency: Currently, all three 'if' statements will be checked every time. You could try something like:

```
if (i%3 != 0 && i%5 != 0){
    //do stuff
}
else {
    if (i%3 == 0){
        //do stuff
    }
    if (i%5 == 0){
        //do stuff
    }
}
```
and then you'd avoid running code that isn't necessary. Pretty small difference in this particular example, but imagine if we ran this for some n = 1,000,000,000,000+ or something like that.  -Nick Hager

Comment (JRSorem) [reply projects/fizzbuzz/java/FizzBuzz.java]: Thanks for the suggestion! I updated it with your ordering.

**Comment by JRSorem** [comment]
> 
> 
> Code works but would like some nicer formatting
> 
> Output:
> 
> Enter a number:
> 17
> 12Fizz4BuzzFizz78FizzBuzz11Fizz1314FizzBuzz1617

Can't reproduce this. It prints out each result on its own line for me.

**Pull request closed without merging**